client/widgets/db-stats/stats.template.html.slim

```diff
@@ -31,26 +29,32 @@
             tr
               th.rows.text-right Count
               th.rows-trend.text-center.db-stats-right-border Trend
-              th.total-size.text-right Total
+              th.total-size.text-right Size
           tbody
-            tr ng-repeat='stat in stats'
+            tr ng-repeat='stat in currentStats'
               td.name.db-stats-right-border {{ stat.name }}
               td.rows.text-right {{ stat.rowsCount | number }}
               td.rows-proportion.text-right.hidden-md.hidden-sm.hidden-xs {{ stat.rowsProportion | number:2 }}%
               td.rows-trend.text-center.db-stats-right-border
                 sparkline ng-if='stat.rowsCountTrend' data='stat.rowsCountTrend'
-              td.table-size.text-right.hidden-md.hidden-sm.hidden-xs {{ stat.tableSize | prettyBytes }}
-              td.indexes-size.text-right.hidden-md.hidden-sm.hidden-xs {{ stat.indexesSize | prettyBytes }}
-              td.total-size.text-right {{ stat.totalSize | prettyBytes }}
+              td.total-size.text-right tooltip='Table: {{ stat.tableSize | prettyBytes }}, Indexes: {{ stat.indexesSize | prettyBytes }}' tooltip-append-to-body='true' {{ stat.totalSize | prettyBytes }}
               td.total-size-proportion.text-right.hidden-md.hidden-sm.hidden-xs {{ stat.totalSizeProportion | number:2 }}%
+            tr.more-tables ng-if='hasMore()'
+              td.text-right.hidden-md.hidden-sm.hidden-xs colspan=6
+                a href='#' ng-click='more()' Show all the {{ stats.length }} tables
```

Comment:
I think it would be better if the text was `Show remaining {{ stats.length - top }} tables`. What do you think?